src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.block {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  display: flex;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.star__active {
```

Comment:
You don't need this class. You need to set stars from container class number with `:nth-child`